frontend/src/services/agent.js

```diff
@@ -89,10 +89,14 @@ const User = {
   getConnected: () => clientWithoutErrorManagement.get('/connecteduser'),
   login: token => client.post('/psychologist/login', { token }),
   sendMail: email => client.post('/psychologist/sendMail', { email }),
-  sendStudentMail: email => clientWithoutErrorManagement.post('/student/sendMail', { email }),
   logout: () => client.post('/psychologist/logout'),
 };
 
+const Student = {
+  sendMail: email => clientWithoutErrorManagement.post('/student/sendMail', { email }),
+  saveAnswer: data => client.post('/student/saveAnswer', data),
```

Comment:
I'm not sure whether I should use `client` or `clientWithoutErrorManagement` here :shrug: